src/Writers/LaTeXWriter.jl

```diff
@@ -404,10 +407,11 @@ function latexinline(io::IO, md::Markdown.Italic)
 end
 
 function latexinline(io::IO, md::Markdown.Image)
+    doc = task_local_storage(:doc)
```

Comment:
I think we could pass `doc` as the last argument to `latex` and `latexinline` everywhere. That would be in line with surrounding code and is basically how we do it `HTMLWriter`. `task_local_storage` is basically global state which I think we'd like to avoid.